Update "Create cherry-pick pull requests automatically" chapter (#241)
pingcap/tidb-dev-guide: src/contribute-to-tidb/cherrypick-a-pr.md

## Troubleshoot cherry-pick

* If there is any error in the cherry-pick process, for example, the bot fails to create some cherry-pick pull requests. You could ask reviewers/committers/maintainers for help.

> * If there are conflicts in the cherry-pick pull requests. You must [resolve the conflicts](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/addressing-merge-conflicts/about-merge-conflicts) to get pull requests merged. Some ways to do this():

Author: ```suggestion
* If there are conflicts in the cherry-pick pull requests. You must [resolve the conflicts](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/addressing-merge-conflicts/about-merge-conflicts) to get pull requests merged. Some ways can solve it:
```

  - Request privileges to the forked repo by sending `/cherry-pick-invite` comment in the cherry-pick pull request if you are a member of the orgnization. When you accepted the invitaion, you could directly push to the pull request branch.
  - Ask committers/maintainers to do that for you if you are not a member of the orgnization.

>   - Manually create a new cherry-pick pull request for the branch .

Author: ```suggestion
  - Manually create a new cherry-pick pull request for the branch.
```
